dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -527,6 +527,7 @@ void FramesDecoder::SeekFrame(int frame_id) {
 
   // Seeking clears av buffers, so reset flush state info
   if (flush_state_) {
+    while (ReadFlushFrame(nullptr, false)) {}
```

Comment:
Why does it help?

Reply:
Apparently `avcodec_flush_buffers` doesn't clean decoder buffers properly anymore, and when we switch to the next file there are frames left in the buffer, and we cannot feed more frames to the decoder without reading them first. So this is an additional mechanism to clean the decoder output buffer as `avcodec_flush_buffers` (or the particular decoder implementation that should do that) doesn't fully work as we expect anymore. When works it should be noop.

Reply:
Thanks!

Reply:
I wish we could narrow it down to a particular change in the codec...